src/frequenz/client/assets/_location.py

```diff
@@ -0,0 +1,50 @@
+# License: MIT
+# Copyright © 2025 Frequenz Energy-as-a-Service GmbH
+
+"""Location information for a microgrid."""
+
+
+import logging
+from dataclasses import dataclass
+from functools import cached_property
+from zoneinfo import ZoneInfo
+
+import timezonefinder
```

Comment:
This should be removed, I think I mentioned but maybe it was confusing. We are dropping the use of `timezonefinder` as they don't provide binaries for arm64 anymore, it is a huge download, and we never used it.

I made these changes in the microgrid API client already but I think I didn't push them yet, I can let you know when they are available in case you want to do a copy&paste.